app/models/deploy_baseline.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+class DeployBaseline
+  def initialize(deploy)
+    @deploy = deploy
+  end
+
+  def total_latency
+    @total_latency ||= begin
+      {
+        pr_production_latency: valid_deployment ? pr_production_latency : 0,
+        staging_production_latency: valid_deployment ? staging_production_latency : 0
+      }
+    end
+  end
+
+  private
+
+  def valid_deployment
+    @deploy&.production && @deploy&.status == "succeeded"
+  end
+
+  def pr_production_latency
+    pull_requests = @deploy&.changeset&.pull_requests || []
+    pull_requests.sum { |pr| @deploy.updated_at.to_i - pr.created_at.to_i }
```

Comment:
should be the average and not the sum ?
deploy 1 pr -> 1 minute
deploy 10 prs -> 10 minutes ... that seems wrong

Reply:
yup, should be average